README.md

```diff
@@ -1,15 +1,100 @@
-# Gene-Expression-Databases
-Repository hosting the latest Gene Expression Databases to be used with PK-Sim.
+# Gene Expression Databases
 
-Find the latest version of the databases in the [releases section](https://github.com/Open-Systems-Pharmacology/Gene-Expression-Databases/releases)
+This repository hosts the latest **Gene Expression Databases** designed for integration with [PK-Sim](https://www.open-systems-pharmacology.org/), a software tool for physiologically-based pharmacokinetic (PBPK) modeling. These databases provide essential gene expression profiles to enhance PBPK simulations and analyses.
 
-## Code of conduct
-Everyone interacting in the Open Systems Pharmacology community (codebases, issue trackers, chat rooms, mailing lists etc...) is expected to follow the Open Systems Pharmacology [code of conduct](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CODE_OF_CONDUCT.md).
+Explore the databases and their application in modeling workflows, particularly for **pharmacologists**, **toxicologists**, and **researchers** aiming to incorporate gene-level insights into their simulations.
 
-## Contribution
-We encourage contribution to the Open Systems Pharmacology community. Before getting started please read the [contribution guidelines](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CONTRIBUTING.md). If you are contributing code, please be familiar with the [coding standards](https://github.com/Open-Systems-Pharmacology/Suite/blob/master/CODING_STANDARDS.md).
+---
+
+## Table of Contents
+- [About](#about)
+- [Getting Started](#getting-started)
+- [Repository Structure](#repository-structure)
+- [Releases](#releases)
+- [Contributing](#contributing)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+- [FAQs](#faqs)
+
+---
+
+## About
+Gene expression data plays a crucial role in PBPK modeling by providing insights into the variability of drug metabolism and transport across different tissues. The curated databases in this repository are optimized for use with PK-Sim and are updated regularly to reflect the latest research.
+
+Key features:
+- High-quality, tissue-specific gene expression data.
+- Seamless compatibility with PK-Sim for streamlined workflows.
+- Regular updates to ensure alignment with the latest scientific findings.
+
+---
+
+## Getting Started
+1. **Download the Databases**:
+   - Visit the [Releases Section](https://github.com/Open-Systems-Pharmacology/Gene-Expression-Databases/releases) to find the latest version.
+   - Download the `.zip` or `.tar.gz` file containing the database.
+
+2. **Usage with PK-Sim**:
+   - After downloading, load the database into PK-Sim by following the instructions in the PK-Sim user manual.
+   - Ensure you are using a compatible version of PK-Sim with the downloaded database.
+
+3. **System Requirements**:
+   - PK-Sim (version X.X or later)
+   - Operating System: Windows/macOS/Linux
+
```

Comment:
System requirements are the same as for PK-Sim/OSP Suite and should not be listed here.